lib/src/http_transport.dart

```diff
@@ -46,7 +46,9 @@ class HttpTransport implements Transport {
       rq.body = request.bodyText!;
     }
     final rs = await _httpClient.send(rq).timeout(_timeout);
-    return Response(rs.statusCode, await rs.stream.bytesToString());
+    final warning = rs.headers['warning'];
```

Comment:
<img width="933" alt="image" src="https://user-images.githubusercontent.com/5561944/230203133-725ea2a7-da22-495c-be1f-9eefc8a32b5e.png">

If there is no key/value in the headers, it returns null.